### ISSUE TITLE
AppImage issues

### ISSUE THREAD

**Issue by edisionnano** [edisionnano/QDiskInfo#5]
- [x] Window has no icon on X11
- [x] MUSL AppImage doesn't work
 ~~The executable inside the zip actions artifact is not executable by default~~
- [x] Won't run on Ubuntu without installing libfuse2
- [ ] Doesn't use Wayland
- [ ] Won't use dark mode even if the system is in dark mode
- [ ] Bundled smartctl doesn't work
- [ ] We need to bundle the Breeze theme
- [ ] Ask for root password only once (perhaps by using a daemon that runs the command with elevated permissions)

**Comment by edisionnano** [comment]
> is there an appimage for this application ? I'd like to see one appimage version!

Yeah, on every commit there's an appimage artifact https://github.com/edisionnano/QDiskInfo/actions/runs/12496742779

**Comment by lucasmz1** [comment]
Thx mate @edisionnano

**Comment by Samueru-sama** [comment]
@edisionnano what's left to fix? I want to fix all the issues so I can add this to [AM](https://github.com/ivan-hc/AM)

**Comment by lucasmz1** [comment]
working on slackware 15, tested by me. very nice, thank you!

**Comment by edisionnano** [comment]
> [@edisionnano](https://github.com/edisionnano) what's left to fix? I want to fix all the issues so that releases are available and I can add this to [AM](https://github.com/ivan-hc/AM)

All the unchecked issues still remain
